NEWS.txt

```diff
@@ -3,6 +3,34 @@ Release Notes
 =============
 
 
+Theano 0.9.0rc3 (6th of March, 2017)
+====================================
+
+This release extends the 0.9.0rc2 and announces the upcoming final release 0.9.
+
+Highlights (since 0.9.0rc2):
+ - Graph clean up and faster compilation
+ - Removed warp-synchronous programming
```

Comment:
add at the end (Make the new GPU back-end give good result with newer CUDA driver)